starport/templates/message/stargate/x/{{moduleName}}/types/message_{{msgName}}_test.go.plush

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsg<%= MsgName.UpperCamel %>_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  Msg<%= MsgName.UpperCamel %>
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: Msg<%= MsgName.UpperCamel %>{
+				Creator: "invalid_address",
```

Comment:
We should wait for https://github.com/tendermint/starport/pull/1456 to be merged in order to use `<%= MsgSigner.UpperCamel %>` instead of `Creator` for this component